updated readme with simple example

orthagonal opened the pull request.

orthagonal committed:
updated readme with simple example

jgallen23 commented on README.md:
```diff
@@ -1,3 +1,62 @@
+    }
+  },
+  // if needed, update each character:
+  (character, results, done) => {
```
this one doesn't take a callback, just to map the data that is returned

jgallen23 replied:
your update logic should be in the auto

orthagonal committed:
updated readme and test

unit test had an extraneous param
readme example updated to demonstrate map and reduce methods better
